.circleci/config.yml

```diff
@@ -49,15 +49,12 @@ jobs:
       - store_artifacts:
           path: miniapp/build/outputs/aar
           destination: aar/
-      - store_artifacts:
-          path: sample/build/outputs/apk
-          destination: sample/
       - persist_to_workspace:
-          root: ~/code
+          root: testapp/build/outputs/apk/debug
           paths:
-            - miniapp/build/
+            - testapp-debug.apk
```

Comment:
Because the build job is common for the `publish-sdk` workflow, should persisting the AAR of SDK be required?

Reply:
It was persisting the `miniapp/build` folder before, but it wasn't working. But maybe it's just because it was missing the `attach_workspace` step in the `publish-sdk` job. I have added `attach_workspace` now and added back in the `persist_to_workspace` for the build folder.